### ISSUE TITLE
[a11][Violation] Accessibility issues reported on RStudio UI using IBM Equal Access Accessibility Checker tool

### ISSUE THREAD

**Issue by nkn2022** [rstudio/rstudio#15319]
### System details

    RStudio Edition : RStudio Server
    RStudio Version : 2024.04.02  Build 764
    OS Version      : RHEL 9
    R Version       : R 4.3.1

### Steps to reproduce the problem

- Install Chrome extension for **IBM Equal Access Accessibility Checker** tool. Ref: https://www.ibm.com/able/toolkit/verify/automated
- Open RStudio UI on Chrome browser 
- Run the tool and we can see that the following `Violation`s are reported.

### Describe the problem in detail

I recently ran the **IBM Equal Access Accessibility Checker** tool on the RStudio UI on a Chrome browser. The results look good overall, only a few things are missing to reach a first formal level of compliance. So kindly help look into these. 

Issues reported were violating **WCAG 2.2** requirements. 
Ref: https://www.ibm.com/able/requirements/checker-rule-sets

Listing of violations categorized based on the checkpoints. 
Full report (includes information on element locations where issue were found): 
[5.1_Accessibility_Report · RStudio Server.xlsx](https://github.com/user-attachments/files/17382753/5.1_Accessibility_Report.RStudio.Server.xlsx)

### `2.4.1 Bypass Blocks`

> **Violation of Rule:** aria_application_labelled
> **Issue:** Element with "application" role does not have a label

> Violation of Rule: aria_banner_label_unique
> Issue: Multiple elements with "banner" role do not have unique labels

### `4.1.2 Name, Role, Value`

> **Violation of Rule:** input_label_exists (Each form control must have an associated label)
> **Issue:** Form control element <textarea> has no associated label

> **Violation of Rule:** element_tabbable_role_valid (A tabbable element must have a valid widget role)
> **Issue(s):**  
> The tabbable element's role 'none' is not a widget role.
> The tabbable element's role 'document' is not a widget role
> The tabbable element's role 'none' is not a widget role.

> **Violation of Rule:** table_aria_descendants (Table structure elements cannot specify an explicit 'role' within table containers)
> **Issue**: 
> An explicit ARIA 'role' is not valid for element within an ARIA role 'table' per the ARIA in HTML specification

### `2.5.8 Minimum Target Size`

> **Violation of Rule:** target_spacing_sufficient (The target must be sufficiently sized or spaced from other targets)
> **Issue:** 
> Undersized target "button" does not have sufficient spacing of 12 CSS pixels from another target "button"
> Undersized target "div" does not have sufficient spacing of 12 CSS pixels from another target "div"

### `2.5.3 Label in Name`

> **Violation of Rule:** label_name_visible
> **Issue:**
> Accessible name does not match or contain the visible label text

### `ARIA specification`

> **Violation of Rule:** aria_attribute_valid (ARIA attributes should be valid for the element and ARIA role to which they are assigned)
> **Issue:**
> The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic" 

### `HTML specification`

> **Violation of Rule:** element_id_unique (Element 'id' attribute values must be unique within a document)
> **Issue:**
> The element has the id "gradientFill-1" that is already in use
> The element has the id "gradientFill-2" that is already in use

### `1.3.1 Info and Relationships`

> **Violation of Rule:** table_headers_exists (Data table must identify headers)
> **Issue:** 
> Table has no headers identified

### Describe the behavior you expected

- [ x ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.posit.co/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@nkn2022 Is this the same as #15320, just with a couple more statements in that one?

**Comment by nkn2022** [comment]
@ronblum I do know how this got created. Let me close this one as duplicate. 
We can track here: https://github.com/rstudio/rstudio/issues/15320
Closing as invalid.
